less/base.less

```
@@ -0,0 +1,431 @@
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  margin-top: 0px;
+  margin-right: 0px;
+  margin-left: 0px;
+  min-width: 400px;
+
+  background-color: var(--body__color);
+
+  font-size: 1.6rem;
+  line-height: 1.6;
+  font-weight: 400;
+  font-family: "system-ui", "sans-serif";
```

You can't wrap all font names in quotation marks:

```suggestion
  font-family: system-ui, sans-serif;
```

If the font name has no spaces, you must not quote it. The text was rendering as Times New Roman before I fixed this locally.

---

Thanks for catching this! Admittedly I was testing in Chrome, so must have not realized it.